test/TEST-46-HOMED/test.sh

```diff
@@ -5,21 +5,25 @@ set -e
 TEST_DESCRIPTION="testing homed"
 
 # Skip the qemu version of the test, unless we have btrfs
-(modprobe -nv btrfs && command -v mkfs.btrfs) || TEST_NO_QEMU=1
+(modprobe -nv btrfs && command -v mkfs.btrfs >/dev/null) || TEST_NO_QEMU=1
 
 # shellcheck source=test/test-functions
 . "${TEST_BASE_DIR:?}/test-functions"
 
+get_bool "${NO_BUILD:-}" && HOMECTL_BIN="homectl" || HOMECTL_BIN="${BUILD_DIR:?}/homectl"
+if ! command -v "$HOMECTL_BIN" >/dev/null; then
+    echo "Built without systemd-homed, skipping the test"
+    exit 0
```

Comment:
don't we have something like exit 77 for these integration tests? or was i hallucinating?

Reply:
Nope, not for the integration tests themselves, because `make` doesn't propagate the code (there's a [PR](https://github.com/systemd/systemd/pull/21833) related to all those shenanigans).